### PR TITLE
Onboarding MSI for daily autogen

### PR DESCRIPTION
**Pull request by serhangursoy-microsoft**

### CODE REVIEW

File: schemas/2015-08-31-preview/Microsoft.ManagedIdentity.json [hunk 1]
```diff
@@ -7,16 +7,6 @@
     "userAssignedIdentities": {
       "type": "object",
       "properties": {
-        "name": {
```
Comment (jorgecotillo) [inline schemas/2015-08-31-preview/Microsoft.ManagedIdentity.json:10]: Are these updates result of running https://github.com/Azure/azure-resource-manager-schemas/actions/workflows/generate-types.yml on your branch?

Comment (jorgecotillo) [reply schemas/2015-08-31-preview/Microsoft.ManagedIdentity.json:10]: nvm, it is.

**Review by jorgecotillo — APPROVED** [review]

**Review by jorgecotillo — CHANGES_REQUESTED** [review]
Can you please the tests locally,, seems like your update is breaking existing tests: https://github.com/Azure/azure-resource-manager-schemas#runing-unit-tests

**Comment by serhangursoy-microsoft** [comment]
Updated the PR. Changed/updated existing test cases. And added two more for newly generated versions. 
Tested locally now - seems to be working as expected

**Review by jorgecotillo — APPROVED** [review]

**Pull request merged**